lib/internal/Magento/Framework/Stdlib/DateTime/Timezone.php

```diff
@@ -174,6 +182,8 @@ public function date($date = null, $locale = null, $useTimezone = true, $include
                     $timeType,
                     new \DateTimeZone($timezone)
                 );
+
+                $date = $this->getDateWithTime($includeTime, $date);
```

Comment:
Can we do

```
if ($includeTime) {
    $date = $this->getDateWithTime($date);
}
```

or it would not pass PHPMD? Because currently when time should not be included with get date without time actually.

Reply:
This not pass because of Cyclomatic Complexity so I check in the function "getDateWithTIme($date)"

Reply:
Ok, then could you please rename this method to `appendTimeIfNeeded` and switch order of parameters `$date, $includeTime`.